packages/deployer/src/handlers/aws/index.ts

```diff
@@ -0,0 +1,54 @@
+import rawConfig from '../../config-data/config.json';
+import * as node from '@airnode/node';
+
+const config = node.config.parseConfig(rawConfig);
+
+function encodeBody(data: node.WorkerResponse): string {
+  return JSON.stringify(data);
+}
+
+export async function startCoordinator() {
+  await node.handlers.startCoordinator(config);
+  const response = { ok: true, data: { message: 'Coordinator completed' } };
+  return { statusCode: 200, body: encodeBody(response) };
+}
+
+export async function initializeProvider(event: any) {
+  // State and config are sent separately
+  const stateWithConfig = { ...event.state, config };
+
+  const [err, initializedState] = await node.promiseUtils.go(node.handlers.initializeProvider(stateWithConfig));
+  if (err || !initializedState) {
+    const msg = `Failed to initialize provider: ${stateWithConfig.settings.name}`;
+    const errorLog = node.logger.pend('ERROR', msg, err);
+    const body = encodeBody({ ok: false, errorLog });
+    return { statusCode: 500, body };
+  }
+
+  const body = encodeBody({ ok: true, data: node.providerState.scrub(initializedState) });
+  return { statusCode: 200, body };
+}
+
+export async function callApi(event: any) {
+  const { aggregatedApiCall, logOptions } = event;
+  const [logs, apiCallResponse] = await node.handlers.callApi(config, aggregatedApiCall);
+  node.logger.logPending(logs, logOptions);
+  const response = encodeBody({ ok: true, data: apiCallResponse });
+  return { statusCode: 200, body: JSON.stringify(response) };
+}
+
+export async function processProviderRequests(event: any) {
+  // State and config are sent separately
+  const stateWithConfig = { ...event.state, config };
```

Comment:
Github doesn't seem to want to let me respond to the original comment since it's now outdated, but yeah this looks good (in both places). The comment above can also go now since the config isn't "sent" anymore

Reply:
Removed the comment. I also removed the AWS handlers from the node package.

Reply:
Cool thanks. I was planning to do that after this was merged anyway 👍🏻